spec/models/services/twitter_spec.rb

```diff
@@ -67,7 +67,8 @@
     end
 
     it "should not cut links when truncating a post" do
-      long_message = SecureRandom.hex(40) + " http://joindiaspora.com/a-very-long-url-name-that-will-be-shortened.html " + SecureRandom.hex(55)
+      long_message = SecureRandom.hex(40) + " http://joindiaspora.com/a-very-long-url-name-that-will-be-shortened.html " +
```

Comment:
Line is too long. [122/120]

Reply:
This can be fixed by formatting these two lines like this:
```ruby
      long_message = SecureRandom.hex(40) +
        " http://joindiaspora.com/a-very-long-url-name-that-will-be-shortened.html " + SecureRandom.hex(195)
```